mkosi/resources/man/mkosi.news.7.md

```diff
@@ -82,16 +82,53 @@
   need to acquire some files for the build process place them somewhere
   sensible within `$BUILDROOT` so that they can be cached when building
   incrementally.
-- When using a tools tree and we use a relaxed sandbox to run a command
-  (qemu, nspawn, ...), we now keep $PATH entries inside the user's home
-  intact. Note that this may cause issues if a PATH entry in your home contains
-  binaries linked against libraries in `/usr` from the host.
-- Introduced new specifier `%I` which resolves to the name of the current
+- When using a tools tree and a relaxed sandbox is used to run a command
+  (qemu, nspawn, ...), we now keep all entries from $PATH outside of
+  `/usr` intact. Note that this may cause issues if a PATH entry
```

Comment:
```suggestion
- When using a tools tree and a relaxed sandbox is used to run a command
  (qemu, nspawn, ...), we now keep all entries from `$PATH` outside of
  `/usr` intact. Note that this may cause issues if a `$PATH` entry
```